components/CheckInOutForm/index.tsx

```diff
@@ -97,7 +100,16 @@ function CheckInOutForm({
         value: String(attr.value),
         color: attr.attribute.color,
       })),
+    textFieldAttributes: inventoryItem?.attributes
+      ?.filter((attr) => !(attr.attribute.possibleValues instanceof Array))
```

Comment:
put the `!(attr.attribute.possibleValues instanceof Array)` condition into the `.reduce`, it'll cut down on array passes